packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -252,12 +252,21 @@ class _LogDetailsState extends State<LogDetails>
   Widget _buildSimpleLog(BuildContext context, LogData log) {
     final disabled = log?.details == null || log.details.isEmpty;
     return OutlineDecoration(
-      child: Console(
-        lines: log?.prettyPrinted?.split('\n'),
-        controls: [
-          CopyToClipboardControl(
-            dataProvider: disabled ? null : () => log?.prettyPrinted,
-            buttonKey: LogDetails.copyToClipboardButtonKey,
+      child: Column(
+        children: [
+          areaPaneHeader(
```

Comment:
Shouldn't this be be the `title` of the `Console` underneath?

Reply:
hmm, will confirm